pages/proposals.jsx

```diff
@@ -2,21 +2,628 @@ var React = require('react');
+          Game
+        </InputCombo>
+        <InputCombo name="format-checkbox" onClick={this.formatCheckboxClicked} className="checkbox-input" for="format-not-sure" type="checkbox">
+          I am not sure
+        </InputCombo>
+        <InputCombo onClick={this.otherFormatCheckboxClicked} name="format-checkbox" isOther={true} className="checkbox-input" for="other-format" type="checkbox">
+          Other
+        </InputCombo>
+        <div id="format-checkboxError" className="error-message">Format is required.</div>
+
+        <label>If you chose "other" above, please outline why.</label>
+        <input onClick={this.otherFormatInputClicked} className="format-checkbox other-input format-other-input" type="text"/>
+        <div id="format-other-error-message" className="error-message">Other cannot be empty.</div>
+
+        <InputCombo wordcount="150" errorMessage="Field is required." for="desc-allow" name="desc-allow" type="textarea">
+          Describe what your session or activity will allow people to make, learn or do in 150 words or less. *
+        </InputCombo>
+
+        <InputCombo wordcount="150" errorMessage="Field is required." for="desc-working" name="desc-working" type="textarea">
+          Describe how you see that working in 150 words or less. *
+        </InputCombo>
+
+        <InputCombo errorMessage="Field is required." for="participants" name="participants" type="text">
+          How will you accommodate varying numbers of participants in your session? Tell us what you'll do with 3 participants? 15? 25? *
+        </InputCombo>
+
+        <InputCombo wordcount="150" errorMessage="Outcome is required." for="outcome" name="outcome" type="textarea">
+          What do you see as outcomes after the festival? How will you and your participants take the learning and activities forward? In 150 words or less. *
+        </InputCombo>
+
+        <InputCombo for="language" name="languge" type="text">
+          Are you comfortable leading your session in another language? If yes, please outline below:
+        </InputCombo>
+
+        <label>MozFest offers limited places for travel sponsorship (stipends) covering flights and accommodation over the festival weekend. These places are offered to those who need support traveling to the event and would have no other means to attend through work or personal covering of costs. We offer these stipends to encourage diversity in facilitators. *</label>
+        <InputCombo className="radio-input" name="travel-radio" for="travel-only" type="radio">
+          I can only attend MozFest if I receive a travel stipend covering my travel and accommodation
+        </InputCombo>
+        <InputCombo className="radio-input" name="travel-radio" for="travel-none" type="radio">
+          I do not require a travel stipend as I, or my work, can cover my costs
+        </InputCombo>
+        <div id="travel-radioError" className="error-message">Travel stipend is required.</div>
```

Comment:
We should probably tweak the error message. `Travel stipend is required.` makes it sound like the proposal will be accepted *only if* Mozilla is paying for the trip.